### ISSUE TITLE
Make CRUD Operations

### ISSUE THREAD

**Issue by Dianadec** [bcgov/reserve-rec-api#45]
#### Description:

Majority of admin will be a mirror of what we offer on our api. - CRUD operations - create/update/delete

Will have this on all of our objects - eg: protected area layer, facility, etc, whatever we are calling them.
In the past in the api side we ignored making crud operations (eg: A&R) - but now it bites us in the butt.

Once confident, they will make api endpoints to facilitate creating/updating/deleting on those layers (sub area area).

#### Acceptance Criteria:
[Note: Use 'Given/When/Then' format if it makes sense to. Otherwise, a simple checklist that can be tested.]

#### Development Checklist:

- [ ] ...
- [ ] ...
- [ ] ...

#### Dependencies

- Blocked by
- Blocking

**Relevant documentation as reference**

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by Dianadec** [comment]
I believe this was a ticket that was then broken out into all the CRUD operation tickets per data type. Thinking we can close this. @cameronpettit  @LindsayMacfarlane . then the overarching Epic can also be closed as completed (https://github.com/bcgov/reserve-rec-api/issues/48)

**Comment by cameronpettit** [comment]
Agreed it can be closed @Dianadec . CRUD operation creation and maintenance is an ongoing process but I think the original intent of this ticket was to get the operations to a first-pass workable state, and only for some of the primary datatypes. More or less we have some preliminary CRUD ops for the following datatypes:

- `protectedAreas`
- `geozones`
- `facilities`
- `activities`
